include/seastar/rpc/rpc.hh

```diff
@@ -623,6 +623,8 @@ public:
             f(*c.second);
         }
     }
+    // Does nothing if there is no connection with the given ID on this server.
+    void abort_connection(connection_id);
```

Comment:
Since now it becomes a public API it deserves a doxygen comment. Or ... (see my other comment regarding client_info)